.factorypath

```diff
@@ -0,0 +1,51 @@
+<factorypath>
```

Comment:
What is this file for though, if it is some local file path change then probably no need to push up

Reply:
I'm also curious about this.

Reply:
+1

Reply:
Im embarrassed to say even i have no clue what this file is. Didn't even realise it got push until you pointed it out... ill add it to .gitignore

Reply:
Oh I think maybe when I switched IDEs, this file was auto generated for some reason..